Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -417,8 +417,8 @@ public struct TransposedConv2D: Layer {
     /// activation function, strides, and padding.
     ///
     /// - Parameters:
-    ///   - filter: The 4-D convolution kernel.
-    ///   - bias: The bias vector.
+    ///   - filter: A 4-D tensor of shape [width, height, inputChannels, outputChannels].
```

Comment:
Identifiers used in this line are not undefined, so I would suggest not avoiding camel case because they are not code. Just use "input channel count" and "output channels count" verbatim.

Reply:
Sorry about that. I was following the python documentation where they use their underscore convention to describe the shape.